### ISSUE TITLE
2 actions sent on radio button click

### ISSUE THREAD

**Issue by pranksinatra** [mike-north/ember-material-lite#59]
I noticed that `{{mdl-radio}}` sends two actions when it is clicked. I extended the component (adding a `change()` and `click()` handler) and realized that it fires a click, then a change, then a click again. Since the component inherits from *mixins/click-action-support*, it sends 2 actions when it is clicked, one for each click event.

Since we're ultimately concerned with radio buttons changing, they should `sendAction` on change instead of click.The change event is [fully supported in IE9+, and all other browsers.](http://www.quirksmode.org/dom/events/change.html)

**Comment by mike-north** [comment]
Thanks for the quick feedback. Will add some tests around this and release another version today

**Comment by mike-north** [comment]
PR #61 addresses your concern. You now have an `on-change` action you can bind to.

```js
test('action "change" test', function(assert) {
  assert.expect(3);

  const counts = {
    tom: 0,
    yehuda: 0,
    stef: 0
  };

  this.on('tomChanged', () => counts.tom++ );
  this.on('yehudaChanged', () => counts.yehuda++ );
  this.on('stefChanged', () => counts.stef++ );

  this.render(hbs`
    {{mdl-radio checked=true value='tom' text='Tom' on-change='tomChanged'}}
    {{mdl-radio value='stef' text='Stefan' on-change='stefChanged'}}
    {{mdl-radio value='yehuda' text='Yehuda' on-change='yehudaChanged'}}
  `);
  
  this.$('input[value="stef"]').click();
  assert.deepEqual(counts, {tom: 0, stef: 1, yehuda: 0}, 'stef: 1');

  this.$('input[value="yehuda"]').click();
  assert.deepEqual(counts, {tom: 0, stef: 1, yehuda: 1}, 'stef: 1, yehuda: 1');

  this.$('input[value="stef"]').click();
  assert.deepEqual(counts, {tom: 0, stef: 2, yehuda: 1}, 'stef: 2, yehuda: 1');

});
```

**Comment by mike-north** [comment]
released in `0.1.13`
